DigitalLearningSolutions.Data/DataServices/RegistrationDataService.cs

```diff
@@ -306,12 +312,16 @@ OUTPUT Inserted.ID
             string? centreSpecificEmail,
             int userId,
             PossibleEmailUpdate? possibleEmailUpdate,
-            IDbTransaction transaction
+            IDbTransaction transaction,
+            DateTime? currentTime = null

```

Comment:
This method is sometimes called in other methods that have already calculated the current time to update other db values. I figured we could just pass it in rather than calculating the current time more than once. It represents the time to update the EmailVerified to, which will be near enough the current time. Is it better to rename it, or just calculate the time again here?

Reply:
I don't think it'll give a noticeable performance boost (DateTime.UtcNow isn't that expensive), so I think it's more worthwhile to make the code simpler and check it every time.